src/sentry/integrations/utils/codecov.py

```diff
@@ -111,6 +111,7 @@ def get_codecov_data(
             )
             tags = {
                 "codecov.request_url": url,
+                "codecov.request_params": params,
```

Comment:
Eventually you can get rid of `_path` and `_ref`.

Ideally we will have a single tag of `url?param1=value1` so we can just click from Sentry's UI.